packages/docusaurus-plugin-content-docs/src/index.ts

```diff
@@ -461,6 +472,16 @@ Available document ids=
           Object.values(content.docsMetadata),
           'version',
         );
+        const rootUrl = getHrefFromSideBar(
+          content.docsSidebars[`version-${versioning.latestVersion}/docs`],
+        );
```

Comment:
Unfortunately the root url is not necessarily the first doc of the sidebar. If the user used a homeDocumentId, we must use it in priority, even if it's not the first doc in the sidebar (even if users usually use the first doc of the sidebar as home documentId).